--- a/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h.115k	2023-03-11 14:55:22.736728439 +0100
+++ b/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h	2023-03-11 14:54:38.956567387 +0100
@@ -45,7 +45,7 @@
 
 // <o SL_CPC_DRV_UART_VCOM_BAUDRATE> UART Baudrate
 // <i> Default: 115200
-#define SL_CPC_DRV_UART_VCOM_BAUDRATE              115200
+#define SL_CPC_DRV_UART_VCOM_BAUDRATE              460800
 
 // <o SL_CPC_DRV_UART_VCOM_FLOW_CONTROL_TYPE> Flow control
 // <usartHwFlowControlNone=> None
